src/bouwdossiers/tests/test_api.py

```diff
@@ -41,9 +57,28 @@ def test_api_malformed_code(self):
 
     def test_api_one_using_stadsdeel_and_dossier(self):
```

Comment:
ik zou `api_one` hernoemen naar `api_detail` (iets meer in lijn met de benaming van DRF: list en detail endpoints).